google-cloud-contrib/google-cloud-nio/pom.xml

```diff
@@ -112,6 +112,18 @@
               <shadedPattern>shaded.cloud-nio.org</shadedPattern>
             </relocation>
             <relocation>
+              <pattern>io</pattern>
```

Comment:
Would we be able to make these shading patterns more specific? By the looks of this, it seems like they might pick up things we don't necessarily want to shade.